scripts/ci/dockerfiles/apache-rat/Dockerfile

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+FROM ibmjava:sfj-alpine
+
+ARG AIRFLOW_APACHERAT_VERSION
+ARG APACHERAT_VERSION
+ARG APACHE_MIRROR="https://www-us.apache.org/dist"
+
+MAINTAINER "Apache Airflow Community <dev@airflow.apache.org>"
```

Comment:
Do we need to change this as a label? Saw in one of your other prs

Reply:
Changed.